nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,38 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.stdenv.mkDerivation {
+      name = "firejail-" + pkg.name;
+      buildCommand = ''
+        mkdir -p $out
+        ln -s ${pkg}/* $out
```

Comment:
You should quote variables like `${pkg}` and `$out` as they may contain spaces (in case of a store path with spaces) and confuse bash.

Reply:
Be careful not to include the glob in the quote though, so `"${pkg}"/*`